.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
The `.idea` folder should not be committed. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer for the way to exclude files related to your IDE setup.

Reply:
same for `.DS_Store`